c2corg_ui/static/js/maincontroller.js

```diff
@@ -78,9 +78,9 @@ app.MainController.prototype.isPath = function(path) {
     return 'home';
   // if topoguide, it can be all kinds of documents
   } else if (path === 'topoguide') {
-    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1
-            || location.indexOf('areas') > -1 || location.indexOf('images') > -1 || location.indexOf('books')
```

Comment:
Was this ``location.indexOf('books')`` (missing ``> -1``) the reason of the problem?
I am a bit in doubt that changing the order of the tests actually fixes it?!